articles/multifactor-authentication/duo/admin-guide.md

```diff
@@ -0,0 +1,136 @@
+# Duo for Administrators
```

Comment:
I feel there is a mix of concepts in this page, some from Guardian and some from Duo (see my comments bellow).